lib/clientv2/ippool.go

```diff
@@ -50,10 +50,13 @@ func (r ipPools) Create(ctx context.Context, res *apiv2.IPPool, opts options.Set
 	}
 
 	out, err := r.client.resources.Create(ctx, opts, apiv2.KindIPPool, res)
-	if out != nil {
-		return out.(*apiv2.IPPool), err
+	if err != nil {
+		return nil, err
 	}
-	return nil, err
+
+	// Enable IPIP globally if required.
```

Comment:
Should we do this before creating the IP Pool? That way we can fail before we successfully create the pool, giving the user another chance.

Otherwise, they might successfully create the pool but it still won't work.

Reply:
Ah yes a good point! Ta.